cli/cmd/integration_aws_s3_channel.go

```diff
@@ -0,0 +1,79 @@
+//
+// Author:: Darren Murray(<darren.murray@lacework.net>)
+// Copyright:: Copyright 2020, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"github.com/AlecAivazis/survey/v2"
+
+	"github.com/lacework/go-sdk/api"
+)
+
+func createAwsS3ChannelIntegration() error {
+	questions := []*survey.Question{
+		{
+			Name:     "name",
+			Prompt:   &survey.Input{Message: "Name: "},
+			Validate: survey.Required,
+		},
+		{
+			Name:     "role_arn",
+			Prompt:   &survey.Input{Message: "Role ARN:"},
+			Validate: survey.Required,
+		},
+		{
+			Name:     "bucket_arn",
+			Prompt:   &survey.Input{Message: "Bucket ARN:"},
+			Validate: survey.Required,
+		},
+		{
+			Name:     "external_id",
+			Prompt:   &survey.Input{Message: "External ID:"},
+			Validate: survey.Required,
+		},
+	}
+
+	answers := struct {
+		Name       string
+		RoleArn    string
+		BucketArn  string
+		ExternalID string
```

Comment:
I think you are missing some annotations here, if not, survey won't know where to put the input variable:

Tested and failed:
```
$ lacework int create
? Choose an integration type to create:  AWS S3 Alert Channel
▸ Name:  name
▸ Role ARN: role
Usage:
  lacework integration create [flags]

Flags:
  -h, --help   help for create

Global Flags:
  -a, --account string      account subdomain of URL (i.e. <ACCOUNT>.lacework.net)
  -k, --api_key string      access key id
  -s, --api_secret string   secret access key
      --debug               turn on debug logging
      --json                switch commands output from human-readable to json format
      --nocolor             turn off colors
      --noninteractive      turn off interactive mode (disable spinners, prompts, etc.)
  -p, --profile string      switch between profiles configured at ~/.lacework.toml

ERROR unable to create integration: could not find field matching role_arn
```